Wastewater_Seq_analysis/run_viralrecon.sh

```diff
@@ -0,0 +1,73 @@
+ #!/bin/bash
+# if any errors, then exit
+set -euxo pipefail
+
+###########################
+# Author: Pooja Gupta
+
+USAGE="
+Purpose: Bash script to run viralrecon bioinformatic pipeline with wastewater sequencing data.
+
+Usage: run_viralrecon.sh <wastewater sequencing run_name>
+
+Last updated on June 13,2022
+"
+###########################
+
+echo $USAGE
+run_name=$1
+
+analysis_dir='/Volumes/IDGenomics_NAS/wastewater_sequencing'
+mkdir -p /Volumes/IDGenomics_NAS/wastewater_sequencing/$run_name/analysis/viralrecon/work
+
+#Creating required folders for analysis
+ww_fastq=$analysis_dir/$run_name/raw_data
+out_dir=$analysis_dir/$run_name/analysis/viralrecon
+work_dir=$analysis_dir/$run_name/analysis/viralrecon/work
+results=$analysis_dir/$run_name/results
+
+infile=${run_name}_samplesheet.csv
+log_file=$out_dir/${run_name}_viralrecon.log
+
+echo "$(date) : Run Wastewater sample data with viralrecon for run $run_name"
+echo "$(date) : First create input samplesheet for viralrecon pipeline"
+
+if [ ! -f "$out_dir/${run_name}_samplesheet.csv" ]
+    then
+        echo "$(date) : $infile does not exist. Creating samplesheet required to run viralrecon"
+        python /home/pgupta/pscripts/fastq_dir_to_samplesheet.py $ww_fastq $out_dir/$infile
```

Comment:
This includes a path that isn't accessible to other users. Can you make the path relative to this script?